watertap/tools/parameter_sweep.py

```diff
@@ -619,7 +617,9 @@ def _write_debug_data(
 
     if write_h5:
         fname_h5 = f"local_results_{rank:03}.h5"
-        _write_output_to_h5(local_results_dict, debugging_data_dir, fname_h5)
+        _write_output_to_h5(
+            local_results_dict, os.path.join(debugging_data_dir, fname_h5)
+        )
     if write_csv:
         fname_csv = f"local_results_{rank:03}.csv"
```

Comment:
Just doing a quick skim, it seems like you removed `write_h5` and `write_csv` arguments, but not sure that you meant to keep them in here.

Reply:
OK- I see what you did here. 

`            h5_results_file_name is not None,
            csv_results_file_name is not None` provide the boolean vals for these in this particular function.

Reply:
Yes -- and in this function it's their existence, not their names, that matter.